apps/web-portal-client/lib/components/layout/Header.jsx

```diff
@@ -1,21 +1,48 @@
-import React from 'react'
-import PropTypes from 'prop-types'
+import React, {useState} from 'react'
+
+import {useAuth} from '../../features/auth/hooks'
+
+import {MenuItem, DesktopMenu, MobileMenu} from './main-menu'
+
+import {Logo} from '../Logo'
+import {Button} from '../Button'
+import {Image} from '../Image'
+import {ImageShapes} from '../ImageShapes'
+
+export function Header() {
+  const [shouldShowMobileMenu, toggleMobileMenu] = useState(false)
+  const {logout} = useAuth()
 
-export function Header(props) {
-  const {title} = props
   return (
-    <header>
-      <div className="mx-auto py-6 px-4 sm:px-6 lg:px-8">
-        <h1 className="text-3xl leading-tight text-gray-900">{title}</h1>
-      </div>
-    </header>
-  )
-}
+    <nav className="bg-gray-800">
+      <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+        <div className="flex items-center justify-between h-16">
+          <div className="flex items-center">
+            <Logo />
 
-Header.propTypes = {
-  title: PropTypes.string,
-}
+            <DesktopMenu>
```

Comment:
DesktopMenu will render without any condition in comparison to MobileMenu?

Reply:
yes

Reply:
It looks strange. We have two types of menu Desktop and Mobile. The names tell me that one of the menus should be shown in case of an open application on a desktop another in case of mobile use. But you tell that desktop is rendered in any case. So what "Desktop" and "Mobile" are mean?

Reply:
view desktop and mobile menu depends on screen size (md:hidden)
But mobile menu can be hidden and shown by click sandwich button unlike desktop menu which shown all the time

Reply:
Why we need a mobile menu on a desktop? This was described in the task?